electron/app/ipc/ipcListeners.ts

```diff
@@ -263,3 +269,75 @@ ipcMain.on('global-electron-store-update', (event, args: any) => {
     log.warn(`received invalid event type for global electron store update ${args.eventType}`);
   }
 });
+
+ipcMain.on('init-shell', (event, args) => {
```

Comment:
let's write the event names in a similar format like you did below
```suggestion
ipcMain.on('shell.init', (event, args) => {
```